Common/TableProducer/trackextension.cxx

```diff
@@ -131,10 +126,26 @@ struct TrackExtension {
     /* when a new run (Run3) is processed                           */
     o2::base::Propagator::MatCorrType matCorr = o2::base::Propagator::MatCorrType::USEMatCorrLUT;
 
+    int lastCollId = -1;
     for (auto& track : tracks) {
       std::array<float, 2> dca{1e10f, 1e10f};
       if (track.has_collision()) {
         if (track.trackType() == o2::aod::track::TrackTypeEnum::Track) {
+          if (lastCollId != track.collisionId()) {
```

Comment:
I don't think this optimization is needed. The one with the run number should be enough...

Reply:
OK!
I will rid of them in both Run2 and Run3